core/drand_status.go

```diff
@@ -25,3 +32,62 @@ const (
 	BeaconNotInited BeaconStatus = iota
 	BeaconInited
 )
+
+func GetDkgStatusDescription(value DkgStatus) string {
+	switch value {
+	case DkgReady:
+		return "Done"
+	case DkgInProgress:
+		return InProgressDesc
+	case DkgNotStarted:
+		return NotStartedDesc
+	default:
+		return UnknownDesc
+	}
+}
+
+func GetReshareStatusDescription(value ReshareStatus) string {
```

Comment:
reshare and DKG share a lot of common logic, like _a lot_ maybe it's not necessary to use a second description just for that ? (open question)

Reply:
Is there any way we can know a reshare process is finished? DKG has a flag to indicate it. Both have different statuses we can recognize, so I preferred to keep them separate.